lib/actions-oclif/os/download.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license
+ * Copyright 2016-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { flags } from '@oclif/command';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { stripIndent } from '../../utils/lazy';
+
+interface FlagsDef {
+	output: string;
+	version?: string;
+	help: void;
+}
+
+interface ArgsDef {
+	type: string;
+}
+
+export default class OsDownloadCmd extends Command {
+	public static description = stripIndent`
+		Download an unconfigured OS image.
+
+		Download an unconfigured OS image for a certain device type.
+		Check available types with \`balena devices supported\`
+
```

Comment:
The Capitano version had these lines in the command help text:
```
> Note: Currently this command only works with balenaCloud, not openBalena.	
> If using openBalena, please download the OS from: https://www.balena.io/os/
```
Accidental or intentional removal?

Reply:
Oops - accidental.

Reply:
I see that those angle brackets are in the source - are they supposed to be there?

Reply:
They were meant to display nicely when formatted as markdown, but feel free to remove.